src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -563,6 +565,24 @@ void ApplyLiteralProperties()
 
 				writer.AppendLineIndented(";");
 			}
+
+			void AttachUnhandledExceptionHandler()
+			{
+				writer.AppendLineIndented($"#if DEBUG");
+				writer.AppendLineIndented($"UnhandledException += (s, e) =>");
+				writer.AppendLineIndented("{");
+				writer.Indent();
```

Comment:
You could do this as `writer.BlockInvariant("UnhandledException += (s, e) =>")`

Reply:
Ignore this comment. `BlockInvariant` may force the semicolon to be in a separate line. We could keep the code more verbose for better formatting